scripts/hal_alignment/maf_to_fasta.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python3
+
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Convert each block of a MAF alignment to a FASTA file.
```

Comment:
```suggestion
"""Convert each block of a multiple alignment format (MAF) alignment to a FASTA file.
```

Reply:
Thanks for this suggestion, this makes it clearer from the get-go what format is being talked about. I would perhaps adjust the wording to refer to a `multiple alignment format (MAF) file` (to avoid repetition). Also, do you have in mind that all the instances of this line should be updated in a similar way?

Reply:
My approach when I code is that in the main module dosctring I define the acronyms, and then later on I only use the acronyms, without more information (similar to what you would do in a paper).